### ISSUE TITLE
FileNotFoundError: [Errno 2] No such file or directory: 'flatpak-builder'

### ISSUE THREAD

**Issue by shakaran** [pitivi/pitivi#144]
pitivi-env fails to install in first time flatpak-builder in Ubuntu 16.04

```
$ source bin/pitivi-env
-> Setting up environment if needed...
Installing org.gnome.Platform

Receiving delta parts: 7/9 1,2 MB/s 138,3 MB/177,9 MB 32 seconds remaining                        Receiving delta parts: 7/9 1,2 MB/s 138,7 MB/177,9 MB 32 seconds remaining                        17 delta parts, 78 loose fetched; 225417 KiB transferred in 192 seconds                           Installing related: org.gnome.Sdk.Locale5 MB/177,9 MB 32 seconds remaining                        Receiving delta parts: 7/9 1,2 MB/s 140,0 MB/177,9 MB 32 seconds remaining                        4 metadata, 1 content objects fetched; 1 KiB transferred in 3 seconds                             Receiving delta parts: 7/9 1,2 MB/s 141,2 MB/177,9 MB 31 seconds remaining                        1 metadata, 0 content objects fetched; 313 B transferred in 2 seconds                             Writing objects: 2                                                                                1 metadata, 0 content objects fetched; 313 B transferred in 2 seconds                             -> Generating myrouteprojects/pitivi-dev/pitivi/build/flatpak/Pitivi.master.json against GStreamer master.Platform.Locale
-> Module: yelp-xsl using online repo: git://git.gnome.org/yelp-xsl
-> Module: yelp using online repo: https://git.gnome.org/browse/yelp                              -> Module: gstreamer using online repo: git://anongit.freedesktop.org/gstreamer/gstreamer         -> Module: gst-plugins-base using online repo: git://anongit.freedesktop.org/gstreamer/gst-plugins-base        
-> Module: gst-plugins-good using online repo: git://anongit.freedesktop.org/gstreamer/gst-plugins-goodving metadata objects: 1/(estimating) 156 bytes/s 313 bytes                                  -> Module: gst-plugins-ugly using online repo: git://anongit.freedesktop.org/gstreamer/gst-plugins-uglyving objects: 28% (7/25) 4,3 kB/s 17,1 kB                                                    -> Module: gst-plugins-bad using online repo: git://anongit.freedesktop.org/gstreamer/gst-plugins-badetadata, 19 content objects fetched; 254 KiB transferred in 6 seconds                          -> Module: gst-libav using online repo: git://anongit.freedesktop.org/gstreamer/gst-libav
-> Module: gst-python using online repo: git://anongit.freedesktop.org/gstreamer/gst-python
-> Module: gst-devtools using online repo: git://anongit.freedesktop.org/gstreamer/gst-devtools   -> Module: gst-editing-services using online repo: git://anongit.freedesktop.org/gstreamer/gst-editing-services
-> Module: pitivi repo: file://myrouteprojects/pitivi-dev/pitivi branch: master
Building Pitivi master 0/17 726,0 kB/s 15,2 MB/329,9 MB 7 minutes 13 seconds remaining            
Traceback (most recent call last):
  File "myrouteprojects/pitivi-dev/pitivi/build/flatpak/pitivi-flatpak", line 631, in <module>
    app_flatpak.run()
  File "myrouteprojects/pitivi-dev/pitivi/build/flatpak/pitivi-flatpak", line 436, in run
    self.setup_dev_env()
  File "myrouteprojects/pitivi-dev/pitivi/build/flatpak/pitivi-flatpak", line 507, in setup_dev_env
    subprocess.check_call(builder_args)
  File "/usr/lib/python3.5/subprocess.py", line 576, in check_call
    retcode = call(*popenargs, **kwargs)
  File "/usr/lib/python3.5/subprocess.py", line 557, in call
    with Popen(*popenargs, **kwargs) as p:
  File "/usr/lib/python3.5/subprocess.py", line 947, in __init__
    restore_signals, start_new_session)
  File "/usr/lib/python3.5/subprocess.py", line 1551, in _execute_child
    raise child_exception_type(errno_num, err_msg)
FileNotFoundError: [Errno 2] No such file or directory: 'flatpak-builder'
```

**Comment by aleb** [comment]
flatpak-builder should be in your PATH. How did you install flatpak?

**Comment by shakaran** [comment]
@aleb yes, it is installed under /usr/bin/flatpak

    $ flatpak --version
    Flatpak 0.6.12

But I don't have any binary called "flatpak-builder", could be a typo? I mean remuve the hyphen and pass "flatpak builder" instead "flatpak-builder"?

**Comment by shakaran** [comment]
It seems that I have missing the flatpak-builder package because it has some conflict between new versions:

flatpak-builder : Depends: flatpak (= 0.6.11-1) but 0.6.12-0alexlarsson1~xenial1

So I remove the old flatpak version and install flatpak-builder. Now is working fine, but the install scripts should check if the binary exist or at least try to install it. In hacking documentation file, there are no instructions for install that packages.

**Comment by aleb** [comment]
I agree FileNotFoundError is not very clear. Interested in making a patch which uses https://docs.python.org/dev/library/shutil.html#shutil.which?

**Comment by shakaran** [comment]
@aleb sorry, I don't have a lot time this week and I don't know where exactly should we make the check, since pitivi-env is a bash script and not python, probably is other interal script calling.

**Comment by aleb** [comment]
We're preparing a release, I'll get to it afterwards.

**Comment by aleb** [comment]
Fixed by dcc513779a3a9df7346368efbf7ca8954fb6447a
